src/pages/Pair2/index.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from 'react'
+import { Button, Input } from 'antd'
+import {
+  ShoppingListHeader,
+  GroupCardsContainer,
+  FullScreenContainer,
+  Quantity,
+  ItemName,
+  QuantityContainer,
+  ItemNameContainer,
+  ActionContainer,
+  ItemAdderContainer,
+} from './styles/Pair2.styled'
+import { MinusOutlined, PlusOutlined } from '@ant-design/icons'
+
+type Props = {
+  list: listEntry[]
+  setList: React.Dispatch<React.SetStateAction<listEntry[]>>
+  id: number
+}
+
+type listEntry = {
+  id: number
+  quantity: number
+  itemName: string
+}
+
+type itemAdder = {
```

Comment:
types should be PascalCase instead of camelCase

Reply:
fixed in commit 9dba039b3202249cd2782d2f05140593bbd22a95